### ISSUE TITLE
fatal error

### ISSUE THREAD

**Issue by ndunand** [ndunand/moodle-mod_choicegroup#98]
I had reports of the following problem from users now using the latest codebase:

```
Warning: Creating default object from empty value in /.../moodle-3.1.1/mod/choicegroup/lib.php on line 896
Notice: Undefined offset: 20554 in /.../moodle-3.1.1/mod/choicegroup/lib.php on line 895 
Fatal error: __clone method called on non-object in /.../moodle-3.1.1/mod/choicegroup/lib.php on line 895
```

seems to be due to code coming from 17c0bb6

**Comment by gedion** [comment]
You reference commit 17c0bb6(https://github.com/ndunand/moodle-mod_choicegroup/pull/90). I didn't work on that commit. I worked on pull/95 and pull/97. I did quick check and my two commits do not seem to be the culprit for this new issue. Regardless, I would love to work on it but my time is constrained at the moment.

**Comment by ndunand** [comment]
Thanks @gedion for having a look at it. My bad, your code is not concerned indeed. I'll work it out, no problem.

**Comment by ndunand** [comment]
Fixed by #102
